plugin_tests/common.py

```diff
@@ -224,6 +235,11 @@ def _postTileViaHttp(self, itemId, fileId):
         self.assertTrue('Item already has' in req.json()['message'] or
                         'Item is scheduled' in req.json()['message'])
 
+        if jobAction == 'delete':
+            jobModel = self.model('job', 'jobs')
+            jobModel.remove(jobModel.find(
+                {}, sort=[('_id', SortDir.DESCENDING)])[0])
```

Comment:
This deletes the most recently created job.  Mongo's _ids increase monotonically, so the sort returns the highest (most recent) first.